### ISSUE TITLE
TypeError: Cannot read property 'content' of undefined for "%" unit

### ISSUE THREAD

**Issue by montgomery1944** [csscomb/csscomb.js#389]
When using CSScomb 3.1.5 on LESS:
```less
.test
{
    width: 100%;
}
```

I am getting exception:

```
TypeError: Cannot read property 'content' of undefined
    at csscomb\lib\options\unitless-zero.js:28:51
    at Object.Node.forEach (csscomb\node_modules\gonzales-pe\lib\node.js:63:17)
    at module.exports.process (csscomb\lib\options\unitless-zero.js:18:14)
    at Object.Node.map (csscomb\node_modules\gonzales-pe\lib\node.js:111:9)
    at csscomb\node_modules\gonzales-pe\lib\node.js:117:22
    at Array.forEach (native)
    at Object.Node.map (csscomb\node_modules\gonzales-pe\lib\node.js:115:22)
    at csscomb\node_modules\gonzales-pe\lib\node.js:117:22
    at Array.forEach (native)
    at Object.Node.map (csscomb\node_modules\gonzales-pe\lib\node.js:115:22)
```

**Comment by LabEG** [comment]
I also have this bug.
But if use not default config, (ex. yandex.json, csscomb.json) all good.
Bug only in default (zen.json) config.

**Comment by stramel** [comment]
I also am having this issue on 3.1.5 on LESS. Running it through atom-csscomb

https://github.com/csscomb/csscomb.js/blob/master/lib/options/unitless-zero.js#L28

**Comment by koistya** [comment]
Having the same issue with LESS files here: [kriasoft/react-starter-kit](https://github.com/kriasoft/react-starter-kit)

**Comment by koistya** [comment]
Also when this error occurs the content of a LESS file is set to an empty string.

**Comment by tonyganch** [comment]
The bug was in Gonzales and it's already been fixed in latest version of the parser.
However I cannot update the package dependency because of compatibility issues.
So I've added a hotfix for this issue and released `csscomb@3.1.6`, please update.

**Comment by stramel** [comment]
@tonyganch Thank you kindly! :+1: :clap:
